docs/UserGuide.adoc

```diff
@@ -78,7 +78,7 @@ Format: `delete <index>`
 === Modify a Recipe — `modify`
 Modifies attributes of a recipe.
 
-Format: `modify <index> [ing/<idx. ingredient, quantity> [ins/instruction; ...] [tags/tag, ...]`
+Format: `modify <index> [ing/<idx. ingredient, quantity> [ins/instruction; ...] [cal/calories] [tags/tag, ...]`
```

Comment:
remove square brackets around ingredient and instructions; these refer to *optional* arguments.

Reply:
isn't modifying it optional ?

Reply:
they are still optional since we may not be modifying the ingredients or instructions.